### PR TITLE
Update configuration for version 5 of `isort`

### PR DESCRIPTION
**Pull request by jugmac00**
- remove deprecated configuration options
- remove duplicate isort check via flake8
- fix sort order of imports

modified:   CHANGES.rst
modified:   setup.cfg
modified:   src/Products/SiteErrorLog/SiteErrorLog.py
modified:   src/Products/SiteErrorLog/tests/testInitialization.py
modified:   tox.ini

### CODE REVIEW

**Review by dataflake — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ language: python
 matrix:
     include:
         - python: "3.6"
-          env: TOXENV=flake8
+          env: TOXENV=lint
+          after_success:
         - python: "2.7"
```
Comment (jugmac00) [inline .travis.yml:8]: Did this line slip in?

Comment (dataflake) [reply .travis.yml:8]: This was intentional.

Comment (jugmac00) [reply .travis.yml:8]: Could you give me a hint about the reasoning?

Comment (dataflake) [reply .travis.yml:8]: If you look at what `after_success` does it should be clear that it doesn't make sense for the linting step. The linting step produces no coverage data and coveralls will misinterpret that as "zero coverage".

**Pull request merged**